modules/data/database_repository.py

```diff
@@ -41,6 +41,9 @@ def on_disconnect(self): ...
     def all(self, limit: int = None, skip: int = None):
         return self._session.query(self._model).limit(limit).offset(skip).all()
 
+    def query(self):
+        return self._session.query(self._model)
+
```

Comment:
This is too low-level.

Reply:
So we need to discuss a different way of doing this. This functionality is pretty important.

Reply:
Moved to #42